docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-datadog.mdx

```diff
@@ -0,0 +1,160 @@
+
+We will configure the Datadog Incident Management plugin to route certain access
+requests to the "Teleport Access" team.
+
+## Step 2/5. Define RBAC resources
+
+Before you set up the Datadog Incident Management plugin, you will need to enable
+Role Access Requests in your Teleport cluster.
+
+(!/docs/pages/includes/plugins/editor-request-rbac.mdx!)
+
+## Step 3/5. Generate Datadog credentials
+
+The Access Request plugin for Datadog receives Access Request events from the Teleport
+Auth Service, formats them into Datadog incidents, and sends them to the Datadog
+API. For this to work, you must provide a valid Datadog API and Application key
+to the Teleport plugin.
+
+### Generate a Datadog API key
+
+First, in the Datadog web UI navigate to **Organization Settings -> Access -> API Keys**
+to view your organization's API keys. To generate a new API key, click on **New Key**
+and name it "Teleport Access". Copy the API key to paste in a later step.
+
+![Datadog dashboard for API Keys](../../../../img/enterprise/plugins/datadog/dashboard-api-keys.png)
+
+### Create a Datadog Service Account
+
+Next, navigate to **Organization Settings -> Identity & Accounts -> Service Accounts**
+to view your organization's service accounts. To create a new service account, click
+on **New Service Account** and name it "Teleport Access". You can provide whatever
+email you'd like to associate with the service account, then assign the role
+"Datadog Standard Role". This role provides the required permissions necessary to
+manage the Teleport Access Request incidents.
+
+![Datadog dashboard for Service Accounts](../../../../img/enterprise/plugins/datadog/dashboard-service-accounts.png)
+
+Now click on the "Teleport Access" service account that was just created and click
+on the **Application Keys** tab and click on **New Key** and name it "Teleport Access"
+to create a new Application key. Copy the Application key to paste in a later step.
+
+![Datadog dashboard for Application Keys](../../../../img/enterprise/plugins/datadog/dashboard-application-keys.png)
+
+## Step 4/5. Enroll the Datadog Incident Management plugin
+
+At this point, you're now ready to enroll the Datadog Incident Management plugin.
+Navigate to **Access Management -> Enroll New Integration -> Datadog**.
+
+![Select enrollment](../../../../img/enterprise/plugins/datadog/select-enrollment.png)
+
+Provide the API and Application keys generated above. Select the desired API endpoint.
+Then provide the Datadog team handle, that you created earlier, as the fallback recipient.
+This should be "teleport-access". The fallback recipient will be the default recipient
+for notifications. You can configure more custom notification routing rules afterwards
+using Access Monitoring Rules.
+
+![Datadog enrollment](../../../../img/enterprise/plugins/datadog/datadog-enrollment.png)
+
+If the recipient is a Datadog team, the team name will be added to the Datadog incident
```

Comment:
> If the recipient is a Datadog team

Can the recipient be something else than a team? If so it would be useful to define supported recipient types.